js/ng-views/show.html

```diff
@@ -0,0 +1,21 @@
+<h2>show page</h2>
+<div>
+	<img src="{{vmShow.post.photo_url}}">
+	<p> {{vmShow.post.author}} </p>
+	<p> {{vmShow.post.body}} </p>
+	<!-- <p>{{$stateParams.id}}</p> -->
+	<a href="#" ng-click="vmShow.post.hide = !vmShow.post.hide">Edit</a>
+	<a href="#">Delete</a>
+</div>
+<div ng-init="vmShow.post.hide = false" ng-show="vmShow.post.hide">
+	<form>
+		<input type="text" placeholder="Photo URL" ng-model="vmShow.post.photo_url"><br>
+		<input type="text" placeholder="vmShow.post.author" ng-model="vmShow.post.author"><br>
```

Comment:
Don't you need to wrap the placeholder value in `{{ }}` so that it is evaluated as Javascript?